modules/spring-config/src/main/java/org/mule/runtime/config/internal/LazyMuleArtifactContext.java

```diff
@@ -555,28 +553,11 @@ private void handleTxManagerFactory(TransactionManagerFactory object) {
   }
 
   private void resetMuleSecurityManager() {
-    boolean registerMuleSecurityManager = false;
-    // Always unregister first the default security manager from Mule.
-    try {
-      getMuleRegistry().unregisterObject(OBJECT_SECURITY_MANAGER);
-      registerMuleSecurityManager = true;
-    } catch (Exception e) {
-      // NoSuchBeanDefinitionException can be ignored
-      if (!hasCause(e, NoSuchBeanDefinitionException.class)) {
-        throw new MuleRuntimeException(createStaticMessage("Error while unregistering Mule security manager"),
-                                       e);
-      }
-    }
-    if (registerMuleSecurityManager) {
```

Comment:
Should this be removed?

Reply:
it is replaced by the proper remove* method calls